Dockerfile

```diff
@@ -17,10 +17,13 @@ RUN set -ex \
     && RUN_DEPS=" \
     libpcre3 \
     mime-support \
-    postgresql-client \
+    postgresql-client-12 \
```

Comment:
It looks like it's currently running version 11.12. Any concerns about updating to 12?
I guess either way now is a good time to do this, since production isn't live yet.